### ISSUE TITLE
Clean up gaprange.py and docs.py.  Update read me

### ISSUE THREAD

**Issue by nmtarr** [SGWilliams/GAPProduction#14]
The modules that were just added back in (gaprange.py and docs.py) have a different code structure than is standard for the repo (the other modules).  I think it's confusing to be inconsistent.  Furthermore, having "docs.py" and "documents.py" is confusing.  Could you make the new functions have the same format as the others and move them to ranges.py and documents.py?

Also, the read me has a list of functions for each module, but the new functions need to be added.

Sorry to be a stickler, but I'm trying to keep things clear and organized.

**Comment by nmtarr** [comment]
On second look, maybe some of the functions are formatted in line with existing modules.  Are all the new functions still operable?

**Comment by SGWilliams** [comment]
Agreed. I was just in a hurry to get my range export script to run. I think
the one function in gaprange.py should be brought into range.py. Same with
docs.py (although I'm unsure exactly what that is doing).

**Comment by SGWilliams** [comment]
Yes, the one function I'm using (gp.gaprange.rngDF), is still working.

From looking at the contents of gapranges.py and ranges.py, it looks like
gaprange.py was the original in GapProd. I may (or may not) have added
rngDF (I can't remember). Either way, I started using it so we need to
either keep it, or fold it into ranges.py.

I will say, there are a few other functions in ranges.py that look like
they would still work given the new dB configuration. Maybe, we should have
left all of GapProd intact, and just edit the functions we needed as we
encountered them (as opposed to removing them from sight).

**Comment by nmtarr** [comment]
My preference would be to move it to ranges.py

It's easy enough to copy functions out of the old version, test them, and paste in the new version right?

**Comment by SGWilliams** [comment]
The only thing that holds me up is there are a number of imports/dependencies on top of gaprange.py and I don't have time to sort that out right now. But eventually.
